lib/web_ui/lib/ui_web/src/ui_web/flutter_views_proxy.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:js_interop';
+import 'package:ui/src/engine.dart';
+
+/// Exposes web-only functionality for this app's `FlutterView`s objects.
+final FlutterViewManagerProxy views = FlutterViewManagerProxy(
+  viewManager: EnginePlatformDispatcher.instance.viewManager,
+);
+
+/// This class exposes web-only attributes for the registered views in a multi-view app.
+class FlutterViewManagerProxy {
+  FlutterViewManagerProxy({
+      required FlutterViewManager viewManager,
+    }) : _viewManager = viewManager;
+  // The proxied viewManager instance.
+  final FlutterViewManager _viewManager;
+
+  /// Returns the `hostElement` configuration value passed from JS when `viewId` was added.
+  ///
+  /// This is useful for plugins and apps to have a safe DOM Element where they
+  /// can add their own custom HTML elements (for example: file inputs for the
+  /// file_selector plugin).
+  JSAny? getHostElement(int viewId) {
+    return _viewManager.getOptions(viewId)?.hostElement as JSAny?;
```

Comment:
is `as JSAny?` necessary? I would expect this to work:
```suggestion
    return _viewManager.getOptions(viewId)?.hostElement;
```

Reply:
```
A value of type 'DomElement?' can't be returned from the method 'getHostElement' because it has a return type of 'JSAny?'. (return_of_invalid_type)
```

It seems like it?